tests/basic/init-env-bad.ref

```diff
@@ -0,0 +1,28 @@
+{
+  "answerElements" : [
+    {
+      "class" : "org.batfish.common.BatfishException$BatfishStackTrace",
+      "answer" : [
+        "org.batfish.common.BatfishException: Batfish job failed",
+        "   at org.batfish.main.Driver$1.run(Driver.java:384)",
+        "Caused by: org.batfish.common.BatfishException: Cannot continue: environment 'env_0a30fa3d-4fd0-4ffc-8ffc-c4a2434b793b' is invalid:",
```

Comment:
darnit, this is probably not going to work for determinism. Is there a way to control the environment name?